packages/web-react/src/components/TextArea/README.md

```diff
@@ -0,0 +1,47 @@
+# TextArea
+
+TextArea enables the user to type in text information. It has textarea, label,
+and an optional message. It could be disabled or have an error state. The label could be hidden
+and show if the textarea is required.
+
+```jsx
+<TextArea id="example" type="text" name="example" isRequired validationState="error" message="validation failed" />
+<TextArea id="example" type="password" name="example" isRequired validationState="error" message="validation failed" />
+<TextArea id="example" type="email" name="example" isRequired validationState="error" message="validation failed" />
```

Comment:
I have problem understand, why `<Textarea />` have `type=""` attribute? it is not described in the documentation below anyway.

Reply:
I think this is the copy&paste bug :-)

Reply:
Yeah, it was. Thanks!